libcudacxx/include/cuda/std/__execution/env.h

```diff
@@ -287,6 +287,12 @@ template <class... _Envs>
 _CCCL_HOST_DEVICE env(_Envs...) -> env<__unwrap_reference_t<_Envs>...>;
 
 #ifndef _CCCL_DOXYGEN_INVOKED
+
+// Partial specialization for no env because NVCC segfaults trying to compile `__tuple<>`
+template <>
+struct _CCCL_TYPE_VISIBILITY_DEFAULT env<>
+{};
```

Comment:
@ericniebler please verify that this is correct

Reply:
this is good.